rotkehlchen/icons.py

```diff
@@ -203,3 +203,18 @@ def add_icon(self, asset: Asset, icon_path: Path) -> None:
             icon_path,
             self.custom_icons_dir / f'{asset.identifier}{icon_path.suffix}',
         )
+
+    def delete_icon(self, asset: Asset) -> Tuple[bool, str]:
+        """Deletes an icon from the filesystem cache."""
```

Comment:
Add more info in the docstring.

Specify the behavior. So that it deletes both coingecko cache and custom set icons for an asset. Also specify what is returned.

On that topic I find the function not really intuitive.

The function is called `delete_icon` and returns the usual `Tuple[bool, str]`, which is meant to return True for success and False with an error in error.

But ... here you can't define success or error properly, since it just tries to delete anything it finds. So I don't think you even need to return anything. We don't do anything with this information, do we?